drivers/sensor/mcp98xx/Kconfig

```diff
@@ -0,0 +1,48 @@
+# MCP98XX temperature sensor configuration options
+
+# Copyright (c) 2016 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig MCP98XX
+	bool "MCP98XX temperature sensor"
+	default y
+	depends on DT_HAS_MICROCHIP_MCP98XX_ENABLED
+	select I2C
+	help
+	  Enable driver for MCP98XX temperature sensor.
```

Comment:
What do you mean by "range"? If you mean "product ranges", I figured that it was in the specific chiptypes (and since most of them are only a singe chip range, listing the chips would be enough I expected.

Reply:
I will add the word "range"